src/kfactory/routing/manhattan.py

```diff
@@ -618,17 +618,15 @@ def route_smart(
         default_start_bundle: list[kdb.Trans] = []
         start_bundles: dict[kdb.Box, list[kdb.Trans]] = defaultdict(list)
 
-        index = 0
         for ts in start_ts:
             p = ts.disp.to_p()
-            b = bboxes[index]
+            b = kdb.Box()
             if b.contains(p):
                 start_bundles[b].append(ts)
             else:
                 for i, b in enumerate(bboxes):
                     if b.contains(p):
                         start_bundles[b].append(ts)
-                        index = i
```

Comment:
Please also remove enumerate here and index assignment, I think they aren't used otherwise